ui/components/src/ContentWithLegend/model/content-with-legend-model.test.ts

```diff
@@ -46,40 +55,152 @@ describe('getContentWithLegendLayout', () => {
   });
 
```

Comment:
The diff on these tests are a little funny because I added a wrapper describe around a bunch of them to get coverage on the size variations too.